QModManager/API/SMLHelper/Assets/PDAItem.cs

```diff
@@ -61,7 +70,7 @@ private void PatchTechDataEntry()
             if (this.UnlockedAtStart)
                 KnownTechHandler.UnlockOnStart(this.TechType);
             else
-                KnownTechHandler.SetAnalysisTechEntry(this.RequiredForUnlock, new TechType[1] { this.TechType }, $"{this.FriendlyName} blueprint discovered!");
+                KnownTechHandler.SetAnalysisTechEntry(this.RequiredForUnlock, new TechType[1] { this.TechType }, Language.main.Get(this.DiscoverMessageKey));
```

Comment:
`Language.main.Get` cannot be used at patch time. It can only be used reliably at run time since we haven't yet patched it with the new usages.  
`SetLanguageLine` only stages lines for patching when the Language `strings` dictionary is initialized (which happens later)

It may be better to simply ensure that the strings sent to `SetAnalysisTechEntry` are being handled correctly for Language Overrides.

Reply:
KnownTech uses the discovery message key as a Language key, so the message returns itself if a key was not found. Changing this to the discover message key should make this work.
(Im a bit tired right now, hopefully what i said makes sense)